content/millennium/r4/general-clinical/condition.md

```diff
@@ -178,8 +178,10 @@ Update an existing condition.
 
 _Implementation Notes_
 
-* Currently `problem-list-item` is supported.
+* Currently `problem-list-item` and `encounter-diagnosis` are supported.
 * Any field which is missing will be interpreted as nulling out or removing data from the resource. See [FHIR<sup>®</sup> Update] for additional details about update operations.
+* `abatementDateTime` is not supported for Conditions with a category of `encounter-diagnosis`.
+* `note` cannot be added if note already exists. An existing note field cannot be modified.
```

Comment:
Is this true of problems as well?

Reply:
This is actually a bit confusing because on our mappings wiki it says "For problems there is nothing in local service to prevent updates" but [here](https://github.com/cerner/fhir.cerner.com/blob/master/lib/resources/r4/condition.yaml#L336) this information was already published. Under condition updates:
<img width="644" alt="Screen Shot 2020-04-03 at 9 25 42 AM" src="https://user-images.githubusercontent.com/27974425/78371239-4cf22e80-758d-11ea-9738-582829547586.png">

Reply:
So for problems, if a particular condition doesn't have a note and the user is sending a note in the update request body, in that case we write note to that particular condition. But if there is already a note available for a condition, then local service will ignore the note from the update request body.

Reply:
Kept note here: https://github.com/cerner/fhir.cerner.com/pull/365/commits/2eb7b4a52698b32cdb3c8d3f9d0af6a11cbafb22#diff-abb99d2139d79f8a2a8f1d214f5f6882R363

Is this correct?

Reply:
+1